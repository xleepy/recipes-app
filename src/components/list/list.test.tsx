import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';
import { List } from '.';

describe('List tests', () => {
  it('should render as ordered list', () => {
    const { asFragment } = render(
      <List title="Lorem ipsum" as="ol">
        <li>test</li>
      </List>
    );

    expect(asFragment()).toMatchSnapshot();
  });

  it('should render as unordered list', () => {
    const { asFragment } = render(
      <List title="Lorem ipsum" as="ul">
        <li>test</li>
      </List>
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
