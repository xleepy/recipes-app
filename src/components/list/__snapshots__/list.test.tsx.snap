// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`List tests > should render as ordered list 1`] = `
<DocumentFragment>
  <div
    class="text-left "
  >
    <h3>
      Lorem ipsum
    </h3>
    <ol>
      <li>
        test
      </li>
    </ol>
  </div>
</DocumentFragment>
`;

exports[`List tests > should render as unordered list 1`] = `
<DocumentFragment>
  <div
    class="text-left "
  >
    <h3>
      Lorem ipsum
    </h3>
    <ul>
      <li>
        test
      </li>
    </ul>
  </div>
</DocumentFragment>
`;
